src/main/java/org/gridsuite/study/server/service/StudyConfigService.java

```diff
@@ -360,4 +363,42 @@ public UUID createGridLayoutFromNadDiagram(UUID sourceNadConfigUuid, UUID cloned
         HttpEntity<DiagramGridLayout> httpEntity = new HttpEntity<>(diagramGridLayout, headers);
         return restTemplate.exchange(studyConfigServerBaseUri + path, HttpMethod.POST, httpEntity, UUID.class).getBody();
     }
+
+    public UUID createDefaultComputationResultFilters() {
+        var path = UriComponentsBuilder.fromPath(DELIMITER + STUDY_CONFIG_API_VERSION + COMPUTATION_RESULT_FILTERS_URI + "/default")
+                .buildAndExpand().toUriString();
+        return restTemplate.exchange(studyConfigServerBaseUri + path, HttpMethod.POST, null, UUID.class).getBody();
+    }
+
+    public UUID getComputationResultFiltersUuidOrElseCreateDefaults(StudyEntity studyEntity) {
```

Comment:
We could manage null values. I don't get why they create this empty json for the spreadsheet config